docs/quickstarts/flink_processor_cli_quickstart.md

```diff
@@ -5,6 +5,13 @@ Commandline Tool to a standalone Flink cluster. The Feathub job simply consumes
 the data from the Flink datagen connector, computes some feature, and prints out the 
 result.
 
+### Prerequisites
+
+In addition to the prerequisites listed in [README](../../README.md#Prerequisites). The
+extra prerequisites to run the quickstart are listed below.
+
+- zip 
```

Comment:
Is this section completed?

Reply:
Specifying a command like `zip` in the prerequisite section does not seem common. Is zip installed by default on Unix like operating system such as Linux and Mac OS? 

I am wondering what is the best practice. Is there any existing project that does this in its prerequisite?

Reply:
zip command is indeed trivial and I can find other projects that add this as a prerequisite, even though it may not be installed by default on some Unix like systems, so I remove it from the prerequisite list.